rfc/async-first-team.md

```diff
@@ -0,0 +1,92 @@
+7. Mitigated Knowledge Loss: With comprehensive documentation and asynchronous communication norms in place, the risk of losing critical knowledge during employee offboarding is significantly reduced. Team members can easily access archived information and refer to previous discussions, ensuring continuity of work and minimizing disruptions caused by personnel changes.
+
+### Disadvantages of Async Team Communication
+
+1. Delayed Responses: Asynchronous communication may lead to delays in receiving responses or feedback, potentially slowing down decision-making processes and project progress.
+2. Lack of Real-Time Collaboration: Unlike synchronous methods, async communication may not facilitate immediate collaboration or brainstorming sessions, which could hinder rapid problem-solving or decision-making.
+3. Miscommunication Risks: Without real-time interaction, there is a higher risk of misinterpretation or misunderstanding of messages, particularly in complex or nuanced discussions.
+4. Coordination Challenges: Coordinating tasks and resolving conflicts may require more effort and time in an async communication environment compared to synchronous methods.
+
+## Async first team process proposed
+
+Our team's characteristics naturally lend themselves to asynchronous communication, yet we recognize the value of synchronous interactions in certain contexts. Synchronous communication becomes particularly beneficial when expediting decision-making, clarifying objectives, and involving a small set of individuals. It's also important to maintain a sense of enjoyment and proportional connections within the team.
+
+We advocate for initiating asynchronous processes, which may culminate in synchronous sessions to reach consensus among team members, starting with a proposal. Following one or more rounds of asynchronous feedback and discussion, a subset of the team CAN be involved on a highly productive synchronous session to align on specific set of items.
+
+For fast final alignments and swift decision-making, it's essential to involve a small subset of the team in high-bandwidth one-on-one or small group synchronous discussions, prioritizing efficiency and flexibility.
+
+Considering the small intersection window of some members of the team, spontaneous synchronous discussions should be encouraged and recognized as they naturally occur, rather than relying solely on scheduled calendar slots that may hinder important conversations. While these spontaneous discussions MAY not always align immediately, it's advisable to plan them with some lead time before critical decisions need to be made.
+
+For sprint planning and ensuring everyone grasps the broader picture, a recurring weekly synchronous meeting may be suggested as a best practice. These meetings should be recorded, have a predefined agenda, and clearly outlined objectives from the outset.
```

Comment:
I find it that a recurring 1/2 meetings per week (as far apart as possible) can be helpful in staying aligned and achieve rapid consensus. More than that typically feels like an interruption.